sdk/keyvault/azkeys/models.go

```diff
@@ -194,14 +194,23 @@ func jsonWebKeyFromGenerated(i *generated.JSONWebKey) *JSONWebKey {
 		return &JSONWebKey{}
 	}
 
+	ops := make([]*Operation, len(i.KeyOps))
+	for j, op := range i.KeyOps {
+		if op == nil {
+			ops[j] = nil
+		} else {
+			ops[j] = to.Ptr(Operation(*op))
```

Comment:
Can't this be simplified to just `ops[j] = op`?  Which then makes me wonder if this `for` loop can just be removed completely.

Reply:
The loop translates `[]*string` to `[]*Operation` because we need the latter type and the two aren't interchangeable. `Operation` does have underlying type `string`; do you mean there's a terser way to do this?

Reply:
Oh I thought that `i.KeyOps` was of type `[]*Operation` was also changed in this PR?

Reply:
`i.KeyOps` here is a generated field of type `[]*string`. `Operation` is a convenience type. This code follows from changing the type of the convenience layer's `JSONWebKey.KeyOps` from `[]*string` to `[]*Operation` (also in this PR).

Reply:
Ah yes I totally missed that, I get it now.

You can simplify this a bit though.

```go
ops[j] = (*Operation)(op)
```

Reply:
Thanks for pointing that out, it's a lot tidier.